src/Abp.EntityFramework/EntityFramework/Repositories/EfRepositoryBaseOfTEntityAndTPrimaryKey.cs

```diff
@@ -98,27 +98,27 @@ public override IQueryable<TEntity> GetAllIncluding(params Expression<Func<TEnti
 
         public override async Task<List<TEntity>> GetAllListAsync()
         {
-            return await GetAll().ToListAsync();
+            return await GetAll().ToListAsync(CancellationTokenProvider.Token);
```

Comment:
Although it is good that this is no longer a breaking change, we should still be allowed to override `cancellationToken` for a specific call as I mentioned in https://github.com/aspnetboilerplate/aspnetboilerplate/pull/5111#discussion_r357471289:

> *While cancelling is usually desirable in a purely Get API, since there is no longer anyone to use the information, we may still want to finish Create/Update/Delete.
See https://stackoverflow.com/questions/50329618/should-i-always-add-cancellationtoken-to-my-controller-actions.*

An alternative is to implement a `Use` scope as I suggested in https://github.com/aspnetboilerplate/aspnetboilerplate/pull/5111#discussion_r357908835.

Reply:
@acjh we can implement it in 5.3, what do you think ?

Reply:
Ok.

Reply:
Created the issue https://github.com/aspnetboilerplate/aspnetboilerplate/issues/5291